promscale/installation/docker.md

```diff
@@ -55,59 +54,9 @@ packages and instructions, see the
 After you have installed Promscale, you can ingest data.
 For instructions, see the [send data][send-data] section.
 
-## Upgrading from the previous alpine image
```

Comment:
We've moved that content to the ugprade guide but maybe we could link to that from here. Mostly because folks upgrading may actually land here before finding the upgrade instructions.

Reply:
Why will users land in install instructions for upgrade instructions? If we add the reference here we need to add it for all installation pages. So IMO we shouldn't reference upgrade-related content on installation instructions. This might confuse the people getting started with Promscale.

Reply:
Ok. We can do some user testing later to see how people do upgrades. Maybe they don't even look at documentation at all in general.

I don't know that I would look for a specific upgrade guide but could be wrong. Maybe upgrading do look at the release notes and we could have a link to the upgrade guide there.